packages/js/src/utils/id-token.test.ts

```diff
@@ -229,23 +229,9 @@ describe('decodeIdToken', () => {
       exp: 2000,
       iat: 1000,
       name: '测试用户',
-      role_names: ['admin'],
     });
   });
 
-  test('decoding valid JWT with wrong payload type should throw', async () => {
-    const { privateKey } = await generateKeyPair('RS256');
-    const jwt = await new SignJWT({ name: null, avatar: undefined, role_names: [123] })
-      .setProtectedHeader({ alg: 'RS256' })
-      .setIssuer('foo')
-      .setSubject('bar')
-      .setAudience('qux')
-      .setExpirationTime(2000)
-      .setIssuedAt(1000)
-      .sign(privateKey);
-    expect(() => decodeIdToken(jwt)).toThrowError(TypeError);
-  });
```

Comment:
We should keep this test case, but exchange the wront typed `role_names` with something else